itest/addrs_test.go

```diff
@@ -205,16 +180,51 @@ func testAddresses(t *harnessTest) {
 	require.NoError(t.t, err)
 }
 
+func sendProof(t *harnessTest, src, dst *tarodHarness, rpcAddr *tarorpc.Addr,
```

Comment:
👍 

Eventually I think we can pass a wrapper around this in as a `proof.Courier`, which'll help us test the various proof transfer options.